msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -52,15 +52,15 @@ The `GetParticipant` API must have a bot registration and ID to generate auth to
 > [!TIP]
 > Get participant IDs and tenant IDs from the [tab SSO authentication](../tabs/how-to/authentication/tab-sso-overview.md).
 
-The `Meeting` API must have `meetingId`, `participantId`, and `tenantId` as URL parameters. The parameters are available as part of the TeamsJS library and bot activity.
+The `Meeting` API must have `meetingId`, `participantId`, and `tenantId` as URL parameters. The parameters are available as part of the TeamsJS and bot activity.
```

Comment:
Oops sorry, I didn't mean to imply we should shorten all instances of "TeamsJS library" with "TeamsJS". Could we retain the library terminology where it makes sense to do so?

Reply:
Sure we'll revert back the changes